serverless.yml

```diff
@@ -3,18 +3,34 @@ service: mes-documents-service
 provider:
   name: aws
   runtime: nodejs8.10
+  environment:
+    NOTIFY_HOSTNAME: http://www.example.com
+    USE_NOTIFY: false
+    IS_LOCAL: true
```

Comment:
could you add the other var's as empty strings fro now :)

NOTIFY_EMAIL_TEMPLATE_ID
NOTIFY_API_KEY
NOTIFY_API_BASE_URL
NOTIFY_POST_TEMPLATE_ID
NOTIFY_EMAIL_WELSH_TEMPLATE_ID

Reply:
I think we could add them when we need them.

Reply:
both of our next sub tasks will require at least the api key- so one of us will be waiting on the other

Reply:
okay, will add that